### PR TITLE
More man page adjustments

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: mkosi.md [hunk 1]
```diff
@@ -1360,15 +1358,15 @@ Format=disk
...
 #!/bin/sh
 cd $SRCDIR
 ./autogen.sh
 ./configure --prefix=/usr
 make -j `nproc`
 make install
 EOF
-# chmod +x mkosi.build
+$ chmod +x mkosi.build
 # mkosi --incremental boot
```
Comment (behrmann) [inline mkosi.md:1370]: That should also work as normal user, no?

**Review by behrmann — APPROVED** [review]

**Comment by DaanDeMeyer** [comment]
The last commit isn't entirely correct, we actually boot with 2 serial consoles, one regular one for the bootloader and a virtio serial console for systemd. But maybe we should go back to a single one and hardcode console=ttyS0 in the kernel arguments.

File: mkosi.md (hunk 1 above)
Comment (keszybz) [reply mkosi.md:1370]: I thought that this uses systemd-nspawn which requires root?

Comment (DaanDeMeyer) [reply mkosi.md:1370]: Exactly this still needs root

**Comment by keszybz** [comment]
> The last commit isn't entirely correct, we actually boot with 2 serial consoles, one regular one for the bootloader and a virtio serial console for systemd. But maybe we should go back to a single one and hardcode console=ttyS0 in the kernel arguments.

I wanted to express that it all appears on the caller's terminal. (What happens inside of the VM is a separate story). Do you have a suggestion how to phrase this better without going into too much detail?

File: mkosi.md (hunk 1 above)
Comment (behrmann) [reply mkosi.md:1370]: My bad, when reading this I mixed it with the line above and read `build` instead of `boot`.

**Comment by behrmann** [comment]
The best solution would be a footnote. A lie that's easier to understand with an accompanying deeper explanation hiding the internals that can be skipped or changed when they change, would maybe be helpful.

**Comment by DaanDeMeyer** [comment]
@keszybz #1506 now contains a commit that drops the virtio-serial console in favor of only a regular serial console. The virtio-serial console is too flaky since the virtio_console module is not built in on every distro and edk2 does not support virtio-serial yet either so it makes debugging early boot stuff too complicated. Once that commit is merged this can go in as is.

**Pull request merged**